sound/soc/intel/common/soc-acpi-intel-lnl-match.c

```diff
@@ -66,22 +66,25 @@ static const struct snd_soc_acpi_endpoint rt712_endpoints[] = {
 };
 
 /*
- * RT722 is a multi-function codec, three endpoints are created for
+ * Endpoints for Realtek multi-function codecs, three endpoints are created for
  * its headset, amp and dmic functions.
  */
-static const struct snd_soc_acpi_endpoint rt722_endpoints[] = {
+static const struct snd_soc_acpi_endpoint rt_sdca_mf_endpoints[] = {
```

Comment:
isn't this a bit too broad assumption? By the name it is covering all RT SDCA codecs.

Reply:
> isn't this a bit too broad assumption? By the name it is covering all RT SDCA codecs.

I think all Realtek SDCA multi function codecs can apply this endpoint array unless there is another endpoint array specific to a codec. @shumingfan Can you confirm it?

Reply:
For now, only jack/mic/amp are in the multi-function codec. But I am not sure if there is any difference with the future multi-function codec.

Reply:
Changed to `jack_amp_dmic_endpoints`